lib/rubygems/specification.rb

```diff
@@ -193,6 +194,12 @@ class Gem::Specification < Gem::BasicSpecification
   @@spec_with_requirable_file = {}
   @@active_stub_with_requirable_file = {}
 
+  # Tracking removed method calls to warn users during build time.
+  REMOVED_METHODS = [:rubyforge_project=].freeze #:nodoc:
```

Comment:
```suggestion
  REMOVED_METHODS = [:rubyforge_project=].freeze # :nodoc:
```

Is that space there significant?

Reply:
Thanks, I'll fix! This change is not intentional.